Source/WebCore/page/NavigatorUAData.cpp

```diff
@@ -40,31 +56,116 @@ Ref<NavigatorUAData> NavigatorUAData::create()
 
 const Vector<NavigatorUABrandVersion>& NavigatorUAData::brands() const
 {
-    return m_brands;
+    static LazyNeverDestroyed<Vector<NavigatorUABrandVersion>> brandVersion;
+    static std::once_flag onceKey;
+    std::call_once(onceKey, [] {
+        Vector<NavigatorUABrandVersion> temp = {
+            NavigatorUABrandVersion {
+                .brand = "AppleWebKit"_s,
+                .version = "605.1.15"_s
+            },
+            NavigatorUABrandVersion {
+                .brand = createArbitraryBrand(),
+                .version = createArbitraryVersion()
+            }
+        };
+
+        auto rng = std::default_random_engine { };
+        std::ranges::shuffle(temp, rng);
+        brandVersion.construct(temp);
+    });
+
+    return brandVersion;
 }
 
 bool NavigatorUAData::mobile() const
 {
+#if PLATFORM(IOS_FAMILY)
+    return true;
+#else
     return false;
+#endif
```

Comment:
We can do better here:

```
#if USE(GLIB)
#include <wtf/glib/ChassisType.h>
#endif
```

```suggestion
#if PLATFORM(IOS_FAMILY)
    return true;
#elif USE(GLIB)
    return chassisType() == WTF::ChassisType::Mobile;
#else
    return false;
#endif
```

Reply:
This is great. I had no idea that existed